src/component/Grid.js

```diff
@@ -3,23 +3,7 @@ import { v4 as uuidv4 } from "uuid";
 import sprite from "../assets/img/sprite.svg";
 import { getTimeFromDate, getParam } from "../utils";
 
-const Grid = ({ time, setTime }) => {
-	// time : 1613000844061
```

Comment:
It always makes me sad when I see commented code except of a comment that describes the logic better. If you wanted to, you could use prop types and enforce these props

Reply:
Or even better, a Typescript interface

Reply:
I will add prop types.